[Ready] Code Bounty: Fake Virus Event

PKPenguin321 opened the pull request:
**What is it**
A new event that will target 1-3 people randomly. The fluff text/emotes from multiple viruses (such as coughing, sneezing, dizziness, confusion, vomiting (just the fluff text no actual vomit), etc) will be thrown at the victims as with a regular virus, but they will not actually be suffering from a virus.

The way it works is that it gives a 3 minute invisible status effect that will throw random spooky messages with increasing spookiness over time (like a real virus would). After the 3 minutes is up, the status effect vanishes and it's like you were never even afflicted. Also, after the 1-3 random people are given this status effect, 1-5 more are made to cough/sneeze/etc exactly once with a randomized brief delay.

**Purpose**
This should hopefully make people stop and think before immediately going to lynch the virologist when they see a "You feel lightheaded!" message. Adds paranoia.

:cl:
add: A new Fake Virus event that makes you think you have a virus when you don't
/:cl:

PKPenguin321 committed:
fake virus event
adds a new fake virus event to trick people into thinking they're sick

Qustinnus commented on code/datums/status_effects/debuffs.dm:
```diff
@@ -740,3 +740,46 @@
+	alert_type = null
+	var/msg_stage = 0//so you dont get the most intense messages immediately
+
+/datum/status_effect/fake_virus/tick()
```
wouldnt it be obvious from the amount of symptoms that this isnt a real virus?

PKPenguin321 replied:
That may be the case and I was going to address it but it's possible that pretty much this whole PR will have to be rewritten anyways should the bounty author want it done differently

PKPenguin321 replied:
After testing a bit more I've decided that it's actually close enough to resemble a real virus that this won't be a big issue

optimumtact commented:
This doesn't seem like it would be fun or interesting at all

Qustinnus commented:
@optimumtact it prevents people from auto-guessing there's a virus if someone coughs. adding multiple causes for events prevents meta

YakumoChen commented:
The original intention was for people to cough or something once or twice to instill paranoia, but I'm honestly not sure if this is better or worse as an event.

Ratvar commented:
"This should hopefully make people stop and think before immediately going to lynch the virologist when they see a "You feel lightheaded!" message. "
... you sure everyday TG player won't just rush and try to kill virologist/chemist/whoever because they are totally infected, and then lie to admins they thought they were infected?

PKPenguin321 commented:
IC issue

Ratvar commented:
But is it a *good* thing to include such issue?

PKPenguin321 commented:
Yes, the game is built from conflict

ATH1909 commented:
> Real talk though, if you're too hasty to even use a medical analyzer on yourself

Stealth 2+ viruses are a thing.

Even as a virologist main, I dunno if this PR is a good idea. Why are we punishing people for reacting appropriately to a potential deathvirus outbreak?

Better idea here: Instead of making your proposed status effect come from a random event, add a new type of hallucination effect (y'know, like fake speech, hallucinatory bubblegum, hallucinatory shocks, etc.) that applies your proposed status effect for a period of time.

Qustinnus commented:
no theres too much hallucinations already so itll be super rare

ATH1909 commented:
> no theres too much hallucinations already so itll be super rare

IMO, there aren't ENOUGH hallucinations currently. You end up seeing repeats often after you play for a while.

Also, if the issue is with it being super rare, you could just... bump up the weighting of the fake disease hallucination. There's a reason why hearing voices is more common than getting the OH YEAH hallucination.

PKPenguin321 committed:
minor single-cough aspect
Now after the 1-3 people are chosen to get the fake virus, 1-5 people are chosen that are forced to cough/sneeze/sniff/headache exactly once after a brief randomized timer.

PKPenguin321 commented:
Latest commit [(as per the bounty author's request)](https://tgstation13.org/phpBB/viewtopic.php?f=5&p=504122#p504094):
>After the 1-3 random people are given this status effect, 1-5 more are made to cough/sneeze/etc exactly once with a randomized brief delay.

ErwinSnowden commented:
give it gbs and rhumba beat fake symptoms to get players made at admins

Dawson1917 commented:
Anything that makes it harder to immediately metagame that there's a sentient disease / rogue viro whenever you sniffle is a good addition to the game

Couls commented:
Why not add it to hallucinations instead?

Tlaltecuhtli commented:
miasma already spawns a bunch of viruses

PKPenguin321 commented:
Can the DNM tag be removed please

MimicFaux commented:
Time for some anecdotal evidence why this PR was a great idea.
Was afflicted by this a couple days ago, was one of the only security available to the crew. The symptoms put me on guard, and their increasing intensity got me worried but I was in mid pursuit of the saboteur. Caught them, and deputized the clown with my hat and pepperspray to deliver the prisoner while I hauled ass to medbay only to stir up a panic among the doctor staff about a stealthy virus.
10/10 classic ss13 gameplay.
